src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -1609,7 +1609,7 @@ class C
         bool x = await [|M|].ContinueWith(a => { return true; }).ContinueWith(a => { return false; });
     }
 }";
-            await TestAsync(text, "global::System.Threading.Tasks.Task<System.Boolean>");
+            await TestAsync(text, "System.Object", testPosition: false);
```

Comment:
See the original bug at https://github.com/dotnet/roslyn/issues/643.  I think there *is* some value is finding Task on some of these types.